internal/postgres/config.go

```diff
@@ -172,16 +172,39 @@ func reloadCommand(name string) []string {
 	// mtimes.
 	// - https://unix.stackexchange.com/a/407383
 	script := fmt.Sprintf(`
+# Parameters for curl when managing autogrow annotation.
+APISERVER="https://kubernetes.default.svc"
+SERVICEACCOUNT="/var/run/secrets/kubernetes.io/serviceaccount"
+NAMESPACE=$(cat ${SERVICEACCOUNT}/namespace)
+TOKEN=$(cat ${SERVICEACCOUNT}/token)
+CACERT=${SERVICEACCOUNT}/ca.crt
+
 declare -r directory=%q
 exec {fd}<> <(:)
 while read -r -t 5 -u "${fd}" || true; do
+  # Manage replication certificate.
   if [ "${directory}" -nt "/proc/self/fd/${fd}" ] &&
     install -D --mode=0600 -t %q "${directory}"/{%s,%s,%s} &&
     pkill -HUP --exact --parent=1 postgres
   then
     exec {fd}>&- && exec {fd}<> <(:)
     stat --format='Loaded certificates dated %%y' "${directory}"
   fi
+
+  # Manage autogrow annotation.
+  # Return size in Mebibytes.
+  size=$(df --human-readable --block-size=M /pgdata | awk 'FNR == 2 {print $2}')
+  use=$(df --human-readable /pgdata | awk 'FNR == 2 {print $5}')
+  sizeInt="${size//M/}"
+  # Use the sed punctuation class, because the shell will not accept the percent sign in an expansion.
+  useInt=$(echo $use | sed 's/[[:punct:]]//g')
+  triggerExpansion="$((useInt > 75))"
+  if [ $triggerExpansion -eq 1 ]; then
+    newSize="$(((sizeInt / 2)+sizeInt))"
+    newSizeMi="${newSize}Mi"
+    d='[{"op": "add", "path": "/metadata/annotations/diskstarved", "value": "'"$newSizeMi"'"}]'
```

Comment:
:question: What do we think about `suggested-pgdata-pvc-size` instead? Too verbose?

Reply:
Per discussion, updated.